src/screensignalingprotocol/ScreenSignalingProtocol.js

```diff
@@ -0,0 +1,199 @@
+/*eslint-disable block-scoped-var, id-length, no-control-regex, no-magic-numbers, no-prototype-builtins, no-redeclare, no-shadow, no-var, sort-vars*/
```

Comment:
I think this is generated by mistake (same as d.ts) as it is for a removed feature.

Reply:
+1 can you uncommit the change?

Reply:
Done!!